vendor/bat-native-ledger/src/bat/ledger/internal/contribution/phase_two.cc

```diff
@@ -514,29 +518,30 @@ void PhaseTwo::PrepareVoteBatch() {
 }
 
 void PhaseTwo::VoteBatch() {
-  braveledger_bat_helper::BatchVotes batch = ledger_->GetBatch();
+  ledger::BatchVotes batch = ledger_->GetBatch();
   if (batch.size() == 0) {
     return;
   }
 
-  braveledger_bat_helper::BATCH_VOTES_ST batch_votes = batch[0];
-  std::vector<braveledger_bat_helper::BATCH_VOTES_INFO_ST> vote_batch;
+  ledger::BatchVotesProperties batch_votes = batch[0];
+  std::vector<ledger::BatchVotesInfoProperties> vote_batch;
```

Comment:
`BatchVotesInfos`?

Reply:
`BatchVotesInfo` should ideally be `BatchVote` and then `vote_batch` can be `batch_votes`, your thoughts?

Reply:
See the previous comment